flutter-gui-tests/res/META-INF/plugin.xml

```diff
@@ -0,0 +1,14 @@
+<idea-plugin>
+  <version>1.0-SNAPSHOT</version>
+  <change-notes>Created the plugin. Added UI Tests for the Flutter plugin.</change-notes>
+  <id>io.flutter.tests.gui.flutter-gui-tests</id>
+  <name>Flutter UI Tests</name>
+  <description>
+    Plugin should be used only for the internal purposes.&lt;br&gt;
```

Comment:
=> Plugin should be used only for internal purposes.

?

Reply:
That is a reminder to not include this plugin in the distribution artifacts.

Reply:
Sorry, I was pointing out grammar.  (But it hardly matters.)